deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts

```diff
@@ -0,0 +1,72 @@
+import { DeploymentManager } from "../../../../plugins/deployment_manager/DeploymentManager";
+import { migration } from "../../../../plugins/deployment_manager/Migration";
+import { exp, proposal } from "../../../../src/deploy";
+
+interface Vars {}
+
+export default migration("1689168483_add_maticx_collateral", {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, vars: Vars) => {
+    const trace = deploymentManager.tracer();
+
+    const maticx = await deploymentManager.existing(
+      "MATICX",
+      "0xfa68FB4628DFF1028CFEc22b4162FCcd0d45efb6",
+      "polygon",
+      "contracts/ERC20.sol:ERC20"
+    );
+    const maticxPricefeed = await deploymentManager.existing(
+      "MATICX:priceFeed",
+      "0x5d37E4b374E6907de8Fc7fb33EE3b0af403C7403",
+      "polygon"
+    );
+
+    const {
+      governor,
+      comet,
+      configurator,
+      cometAdmin,
+    } = await deploymentManager.getContracts();
+
+    const newAssetConfig = {
+      asset: maticx.address,
+      priceFeed: maticxPricefeed.address,
+      decimals: await maticx.decimals(),
+      borrowCollateralFactor: exp(0.55, 18),
+      liquidateCollateralFactor: exp(0.65, 18),
+      liquidationFactor: exp(0.9, 18),
+      supplyCap: exp(6_000_000, 18),
```

Comment:
comp.xyz forums are down right now so I can't check, but are these values the ones recommended by Gauntlet?

Reply:
yes